src/lattice.cpp

```diff
@@ -18,10 +18,13 @@ namespace openmc {
 // Global variables
 //==============================================================================
 
-std::vector<Lattice*> lattices;
+namespace model {
 
+std::vector<Lattice*> lattices;
```

Comment:
Recognizing this is not in scope for what the PR is trying to accomplish, but should this instead be `std::vector<std::unique_ptr<Lattice*>>` (or `shared_ptr`, depending)?  Then we wouldn't need to clean up the lattice instances when done.

Reply:
Yes, I agree that should probably be a unique_ptr -- we can handle that in a separate PR.